controlpanel/frontend/forms.py

```diff
@@ -176,20 +193,64 @@ def __init__(self, *args, **kwargs):
 
 class CreateParameterForm(forms.Form):
     key = forms.CharField(
-        validators=[RegexValidator(r'[a-zA-Z0-9_]{1,50}')]
+        max_length=50,
+        validators=[
+            RegexValidator(
+                r'[a-zA-Z0-9_]{1,50}',
+                message="Must contain only alphanumeric characters and underscores",
```

Comment:
Nice ☺️ - I'd also mention the character limit to avoid any confusion (as in someone entering a 51 chars parameter which looks valid but it's not because of its length)